tests/cursorid/cursorid-serialization-002.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+MongoDB\Driver\CursorID serialization on 32-bit systems
+--SKIPIF--
+<?php if (4 !== PHP_INT_SIZE) { die('skip Only for 32-bit platform'); } ?>
```

Comment:
This needs two separate tests for 32-bit and 64-bit systems due to the different serialisation formats. I thought this was better than writing an `EXPECTF` section to account for both int and string outputs of the 64-bit value.

Reply:
Is it the serialisation formats that differ, or just the debug output?

If it's just the debug output, we do have prior art for using an EXPECTF in [tests/functional/cursorid-001.phpt](https://github.com/mongodb/mongo-php-driver/blob/master/tests/functional/cursorid-001.phpt#L35) that you could copy in.

Reply:
Ah, yeah, that was what I wanted to avoid, but if we already use it then we should also use that in the serialisation tests.